src/app-layout/visual-refresh-toolbar/multi-layout.ts

```diff
@@ -61,6 +61,9 @@ function mergeProps(ownProps: SharedProps, additionalProps: ReadonlyArray<Partia
       toolbar.splitPanelToggleProps = props.splitPanelToggleProps;
       toolbar.onSplitPanelToggle = props.onSplitPanelToggle;
     }
+    if (props.breadcrumbs && !checkAlreadyExists(!!toolbar.hasBreadcrumbsPortal, 'hasBreadcrumbsPortal')) {
+      toolbar.hasBreadcrumbsPortal = true;
+    }
```

Comment:
Does it fix anything? it seems like this property is unused

Reply:
Yes, currently when both navigation and tools are hidden, but you have breadcrumbs, the toolbar does not display at all.

```
<AppLayout navigationHide={true} toolsHide={true} breadcrumbs="test" />
```

Reply:
I am not getting this. The value us never used: https://github.com/search?q=repo%3Acloudscape-design%2Fcomponents%20hasBreadcrumbsPortal&type=code